virtcontainers/vm.go

```diff
@@ -148,9 +215,24 @@ func (v *VM) Start() error {
 	return v.hypervisor.startSandbox()
 }
 
+// Disconnect agent and proxy connections to a VM
+func (v *VM) Disconnect() error {
+	v.logger().Info("kill vm")
+
+	if err := v.agent.disconnect(); err != nil {
+		v.logger().WithError(err).Errorf("failed to disconnect agent")
```

Comment:
`s/Errorf/Error/` as the last param is a simple string (same comment for below).